native-maven-plugin/build.gradle.kts

```diff
@@ -45,7 +45,8 @@ import org.gradle.util.GFileUtils
 plugins {
     `java-library`
     groovy
-    checkstyle
+    // TODO: forced to remove this to allow building on my machine. Remove this before merging.
+//    checkstyle
```

Comment:
There are a few instances where I had to uncomment code to make it build on my machine. If anyone has resolutions for avoiding this, please let me know. In either case, they will of course be removed before merging.

Reply:
Please let me know the errors you are seeing, there shouldn't be any need for commenting these out.

Reply:
Using Java 11 installation for compilation and GraalVM 17 for testing resolved all these issues ✅